docs/guides/src/main/server/importExport.adoc

```diff
@@ -0,0 +1,101 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+
+<@tmpl.guide
+    title="Importing and Exporting Realms"
+    summary="An overview about how to import and export realms">
+
+In this guide, you are going to understand the different approaches for importing and exporting realms using JSON files.
+
+== Exporting a Realm to a Directory
+
+To export a realm, you can use the `export` command.
+
+<@kc.export parameters="--help"/>
+
+To export a realm to a directory, you can use the `--dir <dir>` option.
+
+<@kc.export parameters="--dir <dir>"/>
+
+When exporting realms to a directory, the server is going to create separate files for each realm being exported.
+
+You are also able to configure how users are going to be exported by setting the `--users <strategy>` option. The values available for this
+option are:
+
+* *different_files*: Users export into different files subject to the maximum number of users per file. This is the default value for this option.
+
+* *skip*: {project_name} skips exporting users.
+
+* *realm_file*:  Users export to the same file with the realm settings. The file is similar to "foo-realm.json" with realm data and users.
+
+* *same_file*:  Users export to the same file but different from the realm file. The result is similar to "foo-realm.json" with realm data and "foo-users.json" with users.
+
+If you are exporting users using the `different_files` strategy, you should also be able to set how many users per file you want by setting the `users-per-file` option. If not specified, the default value is `50`.
+
+<@kc.export parameters="--dir <dir> --users different_files --users-per-file 100"/>
+
+== Exporting a Realm to a File
+
+To export a realm to a file, you can use the `--file <file>` option.
+
+<@kc.export parameters="--file <file>"/>
```

Comment:
when i use e.g. `kc.sh export --file=realm.exe` this works and creates a theoretically executable file on my filesystem. can we add a convention that the filename to specify has to end with `json`?

Reply:
IMO, unnecessary.

Reply:
not hard on this, so works for me